pkg/query/api/v1.go

```diff
@@ -212,6 +230,7 @@ func (api *API) parsePartialResponseParam(r *http.Request) (enablePartialRespons
 	const partialResponseParam = "partial_response"
 	enablePartialResponse = api.enablePartialResponse
 
+	// Overwrite the cli flag when provided as a query parameter.
```

Comment:
Did you mean to put this below?

Reply:
Anyway I would be fine with just what we have in doc, so maybe we can drop this comment?